public/pages/EditFeatures/containers/EditFeatures.tsx

```diff
@@ -278,12 +343,20 @@ export function EditFeatures(props: EditFeaturesProps) {
                   </ContentPanel>
                 </EuiPageBody>
               </EuiPage>
+              <EuiPage>
+                <EuiPageBody>
+                  <ContentPanel title="Advanced Settings" subTitle={renderAdvancedSettingsToggle()}>
```

Comment:
Should we show some description below the `Advanced Settings`?

Reply:
Do you mean in addition to the description below in the window size configuration. Currently, the window size config is the only thing in the Advanced settings section, so I'm not sure it makes sense to add a description both under the Advanced settings and under the window size config.

Reply:
Make sense, let's keep it as is.